Project-Euler/Problem022.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Problem 22 - Names scores
+ * @see {@link https://projecteuler.net/problem=22}
+ *
+ * Using names.txt (right click and 'Save Link/Target As...'), a 46K text file containing over
+ * five-thousand first names, begin by sorting it into alphabetical order. Then working out the
+ * alphabetical value for each name, multiply this value by its alphabetical position in the list to
+ * obtain a name score.
+ *
+ * For example, when the list is sorted into alphabetical order, COLIN,
+ * which is worth 3 + 15 + 12 + 9 + 14 = 53, is the 938th name in the list.
+ * So, COLIN would obtain a score of 938 × 53 = 49714.
+ *
+ * What is the total of all the name scores in the file?
+ */
+
+export function namesScores (namesTxt) {
+  namesTxt = namesTxt.replaceAll('"', '')

```

Comment:
This is dirty - there should be no quotes in the input file?

Reply:
No, because the input file is with quotes

Reply:
Nevermind, you are right. Just stripping the quotes still feels slightly hacky to me though :P